examples/spark/images/base/core-site.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+
+<configuration>
+  <property>
+    <name>fs.gs.impl</name>
+    <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem</value>
```

Comment:
Are you sure? There are [multiple](https://www.concur.com/blog/en-us/spark-mesos-on-google-cloud) [sites](https://cloud.google.com/hadoop/google-cloud-storage-connector) that seem to indicate this configuration simply makes the `gs://` URI implementation known.